tests/pytests/common.py

```diff
@@ -595,3 +596,12 @@ def get_passphrase():
     passphrase = get_passphrase() if with_pass else None
 
     return cert_file, key_file, ca_cert_file, passphrase
+
+# Use FT.* command to make sure that the module is loaded and initialized
+def verify_shard_init(env, shard=None):
```

Comment:
I still think it would be cleaner to have a debug command for force connection initialization ..